modules/lightning_features/lightning_media/src/BundleResolverBase.php

```diff
@@ -58,35 +53,17 @@ public static function create(ContainerInterface $container, array $configuratio
   }
 
   /**
-   * Returns all possible bundles for the field type(s) this plugin supports.
-   *
-   * @return MediaBundleInterface[]
-   *   Applicable media bundles, keyed by ID.
+   * {@inheritdoc}
    */
-  protected function getPossibleBundles() {
+  public function getPossibleBundles() {
     $plugin_definition = $this->getPluginDefinition();
 
     $filter = function (MediaBundleInterface $bundle) use ($plugin_definition) {
-      $field = $this->getSourceField($bundle);
+      $field = $this->getSourceFieldForBundle($bundle);
```

Comment:
I added this new method to the `SourceFieldTrait` and removed the locally defined method `getSourceField` below